docs/user-guide/timeouts/timeouts-main.md

```diff
@@ -227,6 +231,24 @@ worth considering no matter which statistic is showing high severity.
 (high-path-count)=
 ### Dealing with a high path count
 
+The number of control flow paths is a major indication of how difficult a rule
+is to solve. Intuitively, in order to obtain a correctness proof for the rule, 
+an argument for the correctness of each of its paths has to be found.
+
+The Certora Prover indicates the path count in the Live Statistics panel of its
+reports. There, the path count is given both globally, and per call.
+
+```{figure} path-count-stats.png
+:name: path-count-stats
+Global and per-call path counts are displayed in the Live Statistics panel for 
+each rule.
+```
+
+In order to reduce the path count of a rule, modularization techniques
+(typically: method summarization) can be applied. However, altering the
```

Comment:
another source of increase of path count is the setup and specifically dispatcher(true) to many alternatives. Can we add a comment on that?

Reply:
happily, always feels a bit piecewise, when I'm listing such things ..

Reply:
done (see above)